pkg/scheduler/framework/plugins/clusteraffinity/types.go

```diff
@@ -28,12 +28,19 @@ import (
 
 	clusterv1beta1 "go.goms.io/fleet/apis/cluster/v1beta1"
 	placementv1beta1 "go.goms.io/fleet/apis/placement/v1beta1"
+	"go.goms.io/fleet/pkg/propertychecker/azure"
 	"go.goms.io/fleet/pkg/propertyprovider"
 )
 
 // clusterRequirement is a type alias for ClusterSelectorTerm in the API, which allows
 // easy method extension.
-type clusterRequirement placementv1beta1.ClusterSelectorTerm
+type clusterRequirement struct {
+	// Embed the original ClusterSelectorTerm.
```

Comment:
we need to backport the https://github.com/kubefleet-dev/kubefleet/pull/330/files first, before updating this file.

Otherwise, backport will have conflicts